R/twas.R

```diff
@@ -452,6 +456,12 @@ mrmash_weights <- function(...) {
     res <- mrmash_wrapper(...)
     return(coef.mr.mash(res)[-1,])
 }
+                   
+#' @importFrom mvsusieR coef.mvsusie
+#' @export               
+mvsusie_weights <- function(mvsusie_fit=NULL,...) {
```

Comment:
This is also wrong ... when mvsusie_fit is null we need to fit mvsusie. Just like this function:

https://github.com/cumc/pecotmr/blob/main/R/twas.R#L390

Otherwise your function by default would not work